sigle/src/modules/analytics/PublishedStoriesFrame.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { SubsetStory } from '../../types';
+import { Box } from '../../ui';
+import { Pagination } from './Pagination';
+import { PublishedStoryItem } from './PublishedStoryItem';
+
+interface PublishedStoriesFrameProps {
+  stories: SubsetStory[];
+}
+
+export const PublishedStoriesFrame = ({
+  stories,
+}: PublishedStoriesFrameProps) => {
+  const [currentStories, setCurrentStories] = useState<SubsetStory[] | null>(
+    stories
+  );
+  const [currentPage, setCurrentPage] = useState<number>(1);
+
+  // how many stories we should show on each page
+  let itemSize = 7;
+
+  useEffect(() => {
```

Comment:
Regarding the `useMemo` question, I don't think we need that optimisation for performance reasons with big lists, but using useMemo here means that we can avoid a setState and avoid a rerender so using it for this sounds good :)

Reply:
oh right I see, thanks for clarifying - will update 👍